Task_TheGridProblem/index.html

```diff
@@ -1,14 +1,24 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Grid Problem</title>
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
-      <a href="../index.html">Go Back</a>
-    </div>
-    <!-- Start coding from the lines below for GRID PROBLEM TASK; DO NOT TOUCH THE CODE ABOVE-->
-  </body>
-</html>
+<!DOCTYPE html>

```

Comment:
Consider giving more descriptive class names than 'one', 'two', 'three', etc., to improve code readability and maintainability.